src/Microsoft.Azure.SignalR/ClientInvocation/TaskCompletionSourceWithCancellation.cs

```diff
@@ -69,31 +73,28 @@ public new void SetException(Exception exception)
 #endif
 
         public static new void SetException(IEnumerable<Exception> exceptions) => Debug.Assert(false);
-        public static new bool TrySetCanceled()
+        public static new void SetCanceled()
```

Comment:
no need to? SetCanceled calls TrySetCanceled internally. Since you make it a general class, the implementation needs to consider potential usage by others.